jujugui/static/gui/src/app/components/user-profile/test-user-profile.js

```diff
@@ -584,95 +566,20 @@ describe('UserProfile', () => {
         charmstore={charmstore}
         env={env}
         getDiagramURL={sinon.stub()}
-        gisf={false}
         interactiveLogin={true}
         pluralize={sinon.stub()}
         storeUser={sinon.stub()}
         user={users.charmstore} />);
     assert.equal(list.callCount, 2);
   });
 
-  it('requests jem envs if jem is provided and updates state', () => {
```

Comment:
Oh and I've moved the specific jem/controller/default tests to https://github.com/juju/juju-gui/pull/1668/files#diff-f609bae29c5075867e5e0facee90cdd8R1435